dcnm/resource_dcnm_inventory.go

```diff
@@ -15,6 +16,8 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+var switchDeployMutexMap = make(map[string]*sync.Mutex, 0)
```

Comment:
This is only used in resource_dcnm_policy it might be better to define it there and not in another file that do not use it.